README.md

```diff
@@ -0,0 +1,19 @@
+# caluma-building-permit-demo
+
+This is a demo project to showcase the usage of caluma, emeis and alexandria in
+a building permit scenario.
```

Comment:
This is outdated, isn't it?

Reply:
Yep

Reply:
Wanna fix it then? ^^

Reply:
Ah i mean outdaten as in it is changed in the next pr, should i just merge the two prs?

Reply:
Don't bother, let's just merge this one.